workflow/rules/common.smk

```diff
@@ -150,7 +150,7 @@ def get_final_output(wildcards):
                         expand(
                             "results/datavzrd-report/{batch}.{event}.{calling_type}.fdr-controlled",
                             batch=get_report_batches(calling_type),
-                            event=get_calling_events(calling_type),
+                            event=event,
```

Comment:
If we remove the use of this utility function here, maybe we can instead use it above, to replace this:
```
for event in config["calling"]["fdr-control"]["events"]:
```
with the function call like this:
```
for event in get_calling_events(calling_type):
```
This ensures, that the calling_type variable from the respective for-loop is also respected, fixing a potential bug.